internal/service/config.go

```diff
@@ -46,28 +50,40 @@ func NewRoomManagerConfig(c config.Config) (room_manager.RoomManagerConfig, erro
 	return roomManagerConfig, nil
 }
 
+// NewWorkersConfig instantiate a new workers Config stucture to be used by the workers to customize them from the config package.
+func NewWorkersConfig(c config.Config) (workers.Configuration, error) {
+	healthControllerExecutionInterval := c.GetDuration(healthControllerExecutionIntervalConfigPath)
```

Comment:
Wdyt about having a default value for this little guy? It could simplify maestro deploy for some users

Reply:
Yeah, I added in the `config/worker.local.yaml`